src/features/campaigns/layout/ProjHomeLayout.tsx

```diff
@@ -102,7 +102,7 @@ const ProjHomeLayout: FC<Props> = ({ children, org, proj }) => {
         >
           <Box sx={{ alignItems: 'center', display: 'flex', gap: 1 }}>
```

Comment:
Last bit of changes - adding the `flexDirection: 'column'` to this `sx`, so that the descripton will lay below the project title, and look a bit better if it is long!